charts/ocean-network-client/README.md

```diff
@@ -48,11 +34,11 @@ helm install my-release spot/ocean-network-client
 | image.pullSecrets | Optional | `[]` | Image pull secrets. |
 | image.repository | Optional | `"public.ecr.aws/spotinst/spot-network-client"` | Image repository. |
 | image.tag | Optional | `""` | Image tag. Defaults to `.Chart.AppVersion`. |
-| namespace | Optional | `"kube-system"` | Namespace where components should be installed. |
+| namespace | string | `"kube-system"` |  |
```

Comment:
@TalShafir @rkrmr33 We have to support different ns for the network-client (since the old controller enabled it, not sure what happens with the new one)

Does the current definition mean that the default ns is kube-system, but we can overwrite it?